### ISSUE TITLE
Not working, same result on two PCs

### ISSUE THREAD

**Issue by lsdxm** [ChristopherHX/mcpelauncher-manifest#129]
Your previously downloaded Minecraft Version might be unsupported or just corrupted.
if you wanted to play a Beta or a new Release please wait patiently for an update,
please choose a compatible version from the profile Editor
otherwise if you have updated the Launcher recently.
e.g. a crash please delete it in Settings,
then download it again via the updated Launcher.

**Comment by ChristopherHX** [comment]
**if you wanted to play** a Beta or **a new Release**
please wait patiently for an update,
**please choose a compatible version from the profile Editor**

~~_You are not trying to launch the new release codename 1.16.20.03_~~.
This works as designed to mitigate a long issue campaign of crashlogs.
**I don't have time to provide the needed update**, please keep using the working 1.16.10.02.
Is this dialog message so confusing? Help me to make it more understandable.

**Comment by ayzenquwe** [comment]
That's quite sad. Many people play on servers that have been updated to the latest version. So the only option now is Boot Camp. :(

**Comment by lsdxm** [comment]
> **if you wanted to play** a Beta or **a new Release**
> please wait patiently for an update,
> **please choose a compatible version from the profile Editor**
> 
> ~_You are not trying to launch the new release codename 1.16.20.03_~.
> This works as designed to mitigate a long issue campaign of crashlogs.
> **I currently don't have the time to provide the needed update**, please keep using the working 1.16.10.02.
> Is this dialog message so confusing? Help me to make it more understandable.
> 
> EDIT:
> 1.16.10 to 1.16.20 changes more to the xbox live interface than 0.12 to 1.16.10.
> Feel free to provide 1.16.20 support by a pull or merge request,
> I'm busy till end of September

Yo I don't need your fucking attitude alright? When people tell you your software isn't working in the form specifically dedicated to doing that, you don't get to attack them like they're some pestering kid nagging you to do them a favor. This is your software, it's not working. Realms has been updated, so the version you're asking me to use is no longer useful for me. I unfortunately am not privy to your lifestyle and schedule, I didn't realize it was such a herculean task for you to bother gracing me with an update.

I can read the dialog. Posting a "do not disturb" sign in your app is not helpful, and neither is your shit attitude that you have every time I post an issue. Fuck you, thanks a lot, now I have to go back to windows because I can't support such an absolute douche bag.

**Comment by chinoto** [comment]
@lsdxm Attitude? He's saying he doesn't have the time to do it, who are you to demand that he works on a project that freely available?
Insulting him won't make this work sooner, in fact you're more likely to make him give up on the project due to entitled users and lack of appreciation.

I would love to get an update because my friend and I have become obsessed with Minecraft recently, but he's on Xbox which means he can't downgrade to a version I can run.
I know some Rust and managed "Hello, World!" in C, so I might be able to figure something out, but learning a new codebase takes time, so I'm not sure if I'd be able to get anything done before @ChristopherHX is free again.

**Comment by nedbert** [comment]
I also am very grateful for the work you have done on this @ChristopherHX .  I'm sure this started as an experiment for you or perhaps you just wanted something so that you could play with your friends.  Most of us really appreciate you taking time to share this application with us.

**Comment by ayzenquwe** [comment]
Yep, @ChristopherHX, thanks a lot for your work. Several my friends were using the launcher without any problems during the quarantine (I think it's actually more than 3 months).

**Comment by ChristopherHX** [comment]
Skilled people can try latest buggy stuff (1.16.20.03 x86_64 and Xboxlive works)
https://github.com/minecraft-linux/mcpelauncher-manifest/issues/374#issuecomment-674563699

**Comment by Olen** [comment]
Just reporting that the flatpak from Edit 5 of https://github.com/minecraft-linux/mcpelauncher-manifest/issues/374#issuecomment-674563699 works fine (at least initially) on Fedora Linux (31).
Guess the Son is going to be happy now!
